doc/source/whatsnew/v0.22.0.txt

```diff
@@ -22,8 +22,8 @@ New features
 Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
-- Better support for ``Dataframe.style.to_excel()`` output with the ``xlsxwriter`` engine. (:issue:`16149`)
-- :func:`pd.tseries.frequencies.to_offset()` now accepts leading '+' signs e.g. '+1h'. (:issue:`18171`)
+- Better support for :func:`Dataframe.style.to_excel` output with the ``xlsxwriter`` engine. (:issue:`16149`)
+- :func:`pandas.tseries.frequencies.to_offset` now accepts leading '+' signs e.g. '+1h'. (:issue:`18171`)
```

Comment:
@jorisvandenbossche @TomAugspurger I can't get this one to render, what is wrong here?

Reply:
Looks like to_offset` isn't in `api.rst`. It would need an

    .. autosummary:
       :toctree: generated/

       tseries.frequencies.to_offset

I think.

Reply:
ahh, and that's actually a pretty useful function, should prob be in ``pd`` namespace